### DISCUSSION TITLE
How to make a custom editing dialogue

### DISCUSSION THREAD

**Discussion by quannm97** [silevis/reactgrid#272] [Q&A]
Hi, I need to create features that when I click on a button, will show a dialog with many inputs, at the footer of the dialog we have 2 buttons to cancel and submit. I can create a custom cell but can't control the state of that cell to toggle the show/hide dialog. Could you please suggest me some solution for this

**Comment by qiufeihong2018** [comment]
If you create a button custom cell, can you show and hide the dialog box through the button?
```js
  render(
    cell: Compatible<OperateBtnCell>,
    isInEditMode: boolean,
    onCellChanged: (cell: Compatible<OperateBtnCell>, commit: boolean) => void,
  ): React.ReactNode {
    return (
      <Button type="primary" text onClick={cell.onClick}>
        {cell.btnText}
      </Button>
    );
  }
```
